test/index.spec.js

```diff
@@ -8,6 +8,28 @@ import awb from './flint-example-awb'
 
 describe('discipl-law-reg', () => {
   describe('The discipl-law-reg library', () => {
+    it('test pegjs generator', async () => {
+      let parsedFact = lawReg.evaluateFactFunction('(NIET [fact1]) OF ([fact2] EN [fact3])')
```

Comment:
Some other testcases we probably want:

- [fact1]  EN [fact2] EN [fact3]
- [fact1]